src/components/CardList.jsx

```diff
@@ -1,47 +1,47 @@
 import styled from 'styled-components';
+import { useEffect, useState } from "react";
 
 function CardList() {
+
+  const apiURL="https://rickandmortyapi.com/api/character";
+  const [characters, setCharacters] = useState([]);
+
+  const Characters =() =>{
```

Comment:
this is a function and should therefore be written in non capital - camel case letters. The name should also represent what the function is doing, e.g. loadCharacters